docs/lms_config.ini

```diff
@@ -32,10 +31,17 @@ progress = SFD_check_progress.html
 
 learneraccountsettings = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-account-settings-page
 
-learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-learner-dashboard
+learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-dashboard
+
+programs = SFD_enrolling.html#programs
```

Comment:
I confirmed with Renzo yesterday that the Programs tab isn't available in Open edX, but having this link won't break anything.